### PR TITLE
docs: add descriptions for workflow argument parameters TDE-1222

### PR DESCRIPTION
**Pull request by amfage**
#### Motivation

Improve the experience for users who submit Argo workflows using the GUI by offering tooltip help text for each workflow input parameter.

#### Modification

Add description fields to all parameters and update existing descriptions with examples, where appropriate.

#### Checklist

- [ ] Tests updated N/A no unit tests for workflows
- [x] Docs updated
- [x] Issue linked in Title

### CODE REVIEW

File: workflows/raster/standardising.yaml [hunk 1]
```diff
@@ -26,16 +26,19 @@ spec:
   arguments:
     parameters:
       - name: version_argo_tasks
+        description: 'Specify a version of the argo-tasks image to use, e.g. "4.1" or "latest"'
```
Comment (blacha) [inline workflows/raster/standardising.yaml:29]:
```suggestion
        description: 'Specify a version of the argo-tasks image to use, e.g. "v4.1" or "latest"'
```

File: workflows/raster/standardising.yaml [hunk 2]
```diff
@@ -26,16 +26,19 @@ spec:
...
         value: 'v4'
       - name: version_basemaps_cli
+        description: 'Specify a version of the basemaps-cli image to use, e.g. "v7.1" or "latest"'
         value: 'v7'
       - name: version_topo_imagery
+        description: 'Specify a version of the topo-imagery image to use, e.g. "4.8" or "latest"'
```
Comment (blacha) [inline workflows/raster/standardising.yaml:35]:
```suggestion
        description: 'Specify a version of the topo-imagery image to use, e.g. "v4.8" or "latest"'
```

File: workflows/raster/standardising.yaml [hunk 3]
```diff
@@ -59,10 +62,13 @@ spec:
           - 'wellington'
           - 'west-coast'
       - name: source
+        description: 'S3 location of the source dataset(s). Separate multiple sources with a semicolon (;) for merging. Target imagery will be layered (bottom to top) in order specified (left to right)'
         value: 's3://linz-imagery-staging/test/sample/'
       - name: include
+        description: 'Regular expression pattern match for paths/files to include e.g ".tiff?$"'
         value: '.tiff?$'
       - name: scale
+        description: 'Scale of the standardised output imagery, e.g. "10000"'
```
Comment (blacha) [inline workflows/raster/standardising.yaml:71]: do we need "eg" when the enum has all the values in it?

File: workflows/raster/standardising.yaml (hunk 2 above)
Comment (amfage) [reply workflows/raster/standardising.yaml:35]: Great catch, thank you!

File: workflows/raster/standardising.yaml (hunk 3 above)
Comment (amfage) [reply workflows/raster/standardising.yaml:71]: Yeah probably not.

File: workflows/raster/publish-odr.yaml [hunk 4]
```diff
@@ -26,12 +26,13 @@ spec:
   arguments:
     parameters:
       - name: version_argo_tasks
+        description: 'Specify a version of the argo-tasks image to use, e.g. "4.1" or "latest"'
```
Comment (paulfouquet) [inline workflows/raster/publish-odr.yaml:29]:
```suggestion
        description: 'Specify a version of the argo-tasks image to use, e.g. "v4.1" or "latest"'
```

File: workflows/raster/standardising.yaml [hunk 5]
```diff
@@ -103,9 +116,10 @@ spec:
         description: '(Optional) location of a cutline file to cut the imagery to .fgb or .geojson'
         value: ''
       - name: collection_id
-        description: '(Optional) If an existing dataset add collection ID here, else a new one will be generated.'
+        description: '(Optional) if an existing dataset add collection ID here, else a new one will be generated.'
```
Comment (paulfouquet) [inline workflows/raster/standardising.yaml:119]: The other `description`s starting with `(Optional) Abc` have a Capital letter for the `Abc`

Comment (blacha) [inline workflows/raster/standardising.yaml:119]:
```suggestion
        description: '(Optional) If an existing dataset add collection ID here, else a new one will be generated.'
```

**Review by blacha — APPROVED** [review]

**Review by paulfouquet — APPROVED** [review]

**Pull request merged**